pulsar_metrics/metrics/utils.py

```diff
@@ -61,3 +59,24 @@ def psi(new: pd.Series, reference: pd.Series, binned: bool = False):
     psi = ((percents["new"] - percents["ref"]) * np.log(percents["new"] / percents["ref"])).sum()
 
     return psi
+
+
+def mmd(new: pd.DataFrame, reference: pd.DataFrame, kernel="linear", **kwargs):
```

Comment:
Minor comment: For better readability you can change def max_mean_dcp() { Maximum Mean Discrepency}

Note: Same comment applicable to other places in the code

Reply:
Done !